dfconvert/make_ipy.py

```diff
@@ -1,28 +1,399 @@
-from collections import defaultdict
-import json
-import os
-from dfconvert.constants import DEFAULT_ID_LENGTH,DF_CELL_PREFIX
-from dfconvert.topological import topological
 import ast
-#Adds tokens to the ast
+import ast_comments
+from io import StringIO
+from .constants import DEFAULT_ID_LENGTH
+import tokenize
+import json
+from typing import Any
+import itertools
 import asttokens
-import IPython.core
-import re
 import astor
-import sys
-
-
-cell_template = {
-   "cell_type": "code",
-   "execution_count": None,
-   "metadata": {},
-   "outputs": [],
-   "source": []
-  }
-
+import builtins
+import re
 
-transformers = []
+ref_uuids = set()
+
+class DataflowRef:
```

Comment:
Please import this from Dfkernel and make dfkernel a dependency, with things changing as they are we should ensure consistent behavior from dfkernel and the best way to do this is by importing from the up to date kernel